### DISCUSSION TITLE
Welcome to linkml Discussions!

### DISCUSSION THREAD

**Discussion by hsolbrig** [linkml/linkml#141] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cmungall** [comment]
Since initiating discussions, we have moved to a mixture of

- community slack
- linkedin group
- monthly community calls
- main github for specific issues

See main site for details

Everyone is still feel free to start new discussions and introduce yourself here! But I'll close this discussion for now
